README.md

```diff
@@ -355,6 +355,24 @@ checkpointing so the job restarts near where it was interrupted.
     --cluster xpk-test --filter-by-job=$USER
     ```
 
+* Workload List supports waiting for the completion of a specific job. XPK will follow an existing job until it has finished or the `timeout`, if provided, has been reached  and then list the job. If no `timeout` is specified, the default value is set to the max value, 1 week. Note: `restart-on-user-code-failure` must be set
+when creating the workload otherwise the workload will always finish with `Completed` status.
+
+  Wait for a job to complete.
+
+    ```shell
+    python3 xpk.py workload list \
+    --cluster xpk-test --wait-for-job-completion=xpk-test-workload
+    ```
+
+  Wait for a job to complete with a timeout of 300 seconds.
+
+    ```shell
+    python3 xpk.py workload list \
+    --cluster xpk-test --wait-for-job-completion=xpk-test-workload \
+    --timeout=300
+    ```
+
```

Comment:
Nit: to add an example or just a sentence that describes the different return codes / scenarios that --wait-for-job-completion can output.

Reply:
Done!